### PR TITLE
DL-3195 Add unit test for avoidance scheme details page

### PR DESCRIPTION
**Pull request by MarjanMubarok**
# JIRA Story or Github Number - descriptive title

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist

*Reviewee* (Replace with your name)
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

*Reviewer* (Replace with your name)
 - [ ]  I've confirmed that every effort has been made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've confirmed appropriate tests has been included with any code added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've confirmed code was added using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: test/views/reliefs/avoidanceSchemes.scala [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  with MockitoSugar with BeforeAndAfterEach with GivenWhenThen with MockAuthUtil {
+
+  implicit val request = FakeRequest()
+  implicit val messages: Messages = app.injector.instanceOf[MessagesApi].preferred(request)
+
+  implicit val mockAppConfig: ApplicationConfig = app.injector.instanceOf[ApplicationConfig]
+  implicit lazy val authContext: StandardAuthRetrievals = organisationStandardRetrievals
+  val periodKey = 2015
+//
+//  @(periodKey: Int, taxAvoidanceForm: Form[TaxAvoidance], serviceInfoContent: Html = HtmlFormat.empty, backLink: Option[String])
```
Comment (Sir-Iyke) [inline test/views/reliefs/avoidanceSchemes.scala:43]: Can this commented out text be deleted?

Comment (MarjanMubarok) [reply test/views/reliefs/avoidanceSchemes.scala:43]: Yeah hahaha. My bad.

File: test/views/reliefs/avoidanceSchemes.scala [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.reliefs
+
+import config.ApplicationConfig
+import forms.ReliefForms._
+import models.{IsTaxAvoidance, ReliefsTaxAvoidance, StandardAuthRetrievals}
```
Comment (Sir-Iyke) [inline test/views/reliefs/avoidanceSchemes.scala:21]: There's imports that aren't being used which can be removed

Comment (MarjanMubarok) [reply test/views/reliefs/avoidanceSchemes.scala]: imports optimised.

**Pull request merged**